### PR TITLE
Update DEVELOPER.md

### PR DESCRIPTION
**Pull request by sakshamtaneja21**
### Summary
fixed some typo 
SUMMARY_GOES_HERE

### Full changelog

* [Implement ...]
* [Add related tests]
* ...

### Issues resolved

Fix #XXX

### CODE REVIEW

File: DEVELOPER.md [hunk 1]
```diff
@@ -1,5 +1,5 @@
 
-These are the steps we follow at Kong to set up a development environment.
+These are the steps we follow in Kong to set up a development environment.
```
Comment (levfishbluefish) [inline DEVELOPER.md]: incorrect change

File: DEVELOPER.md [hunk 2]
```diff
@@ -117,7 +117,7 @@ These commands don't have to be performed as root, since all compilation is done
         --pcre 8.43
 ```
 
-After this task, we'd like to have the next steps use the built packages and for LuaRocks to install new packages inside this `build` directory.  For that, it's important to set the `$PATH` variable accordingly:
+After this task, we'd like to have the next steps to use the built packages and for LuaRocks to install new packages inside this `build` directory.  For that, it's important to set the `$PATH` variable accordingly:
```
Comment (levfishbluefish) [inline DEVELOPER.md]:
```suggestion
After this task, the following steps will use these built packages and LuaRocks will install new packages inside this `build` directory.  It is important to set the `$PATH` variable accordingly:
```

**Comment by sakshamtaneja21** [comment]
done

File: DEVELOPER.md (hunk 2 above)
Comment (sakshamtaneja21) [reply DEVELOPER.md]: done

**Comment by hishamhm** [comment]
@sakshamtaneja21 Could you go through the [https://cla-assistant.io/Kong/kong?pullRequest=5183](CLA Assistant process) so we can accept your PR? thank you!

**Comment by kikito** [comment]
I am closing this PR down since we could not get a CLA signature. Please reopen if you decide to go with it later on, @sakshamtaneja21 .

**Comment by sakshamtaneja21** [comment]
please check @kikito  i have done with CLA

**Pull request closed without merging**